packages/kit/src/exports/vite/index.js

```diff
@@ -379,6 +379,8 @@ function kit() {
 				);
 
 				log.info('Building server');
+				const assets_prefix = `${svelte_config.kit.paths.base}/${svelte_config.kit.appDir}/`;
+				log.info(`assets_prefix: ${assets_prefix}`);
```

Comment:
this seems like a temporary debug line that we probably don't need. and then you could inline `assets_prefix`
```suggestion
```